packages/components/src/components/loaders/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
nit: no need to import React if you don't use it

Reply:
e1b9c0e5d6